cohortextractor/patients.py

```diff
@@ -530,3 +530,31 @@ def household_as_of(reference_date, returning=None, return_expectations=None):
         ),
     """
     return "household_as_of", locals()
+
+
+def attended_emergency_care(
+    on_or_before=None,
+    on_or_after=None,
+    between=None,
+    returning="binary_flag",
+    find_first_match_in_period=None,
+    find_last_match_in_period=None,
+    date_format=None,
+    with_these_diagnoses=None,
+    discharged_to=None,
+    return_expectations=None,
+):
+    """Return information about attendance of A&E.
+
+    Options for `returning` are:
+
+        binary_flag: Whether patient attended A&E
+        date_arrived: date patient arrived in A&E
```

Comment:
I see we have `date_admitted` (for ICU api) as a `returning` option already - should we use that name?

Reply:
Helen says that "admitted" might imply that a patient was admitted to hospital from A&E, so I wanted to avoid that term.

Reply:
OK!

Should we include a summary of the discussions from the issue in the docstring?  (At least link to them I guess)

Reply:
Good idea, I'll update.